terraform/environment-pipelines/iam.tf

```diff
@@ -669,8 +669,8 @@ data "aws_iam_policy_document" "postgres" {
         "lambda:GetLayerVersion"
       ]
       resources = [
-        "arn:aws:lambda:eu-west-2:763451185160:layer:python-postgres:1",
-        "arn:aws:lambda:eu-west-2:763451185160:layer:python-requests:8"
+        "arn:aws:lambda:eu-west-2:763451185160:layer:python-postgres:*",
```

Comment:
My understanding was that cross account access does not allow the `ListLayers` action to be called to retrieve layer version dynamically, so we hardcoded it.